src/models.ts

```diff
@@ -11,11 +11,19 @@ export enum ClimbState {
 
 // ClimberInfo contains information about a climber.
 export class ClimberInfo {
+  initials: string;
+
   constructor(
     public name: string,
     public states: Record<string, ClimbState>,
     public color: string
-  ) {}
+  ) {
+    this.initials = name
+      .split(/\s+/, 2)
```

Comment:
Consider what happens for someone with a space in their first name, such as Amy Jo Kim.

Reply:
three characters easily fit in the button, so i've changed the 2 to 3 here